src/amo/actions/reviews.js

```diff
@@ -39,6 +38,9 @@ export const SEND_REPLY_TO_REVIEW: 'SEND_REPLY_TO_REVIEW' =
   'SEND_REPLY_TO_REVIEW';
 export const SEND_REVIEW_FLAG: 'SEND_REVIEW_FLAG' = 'SEND_REVIEW_FLAG';
 export const UPDATE_ADDON_REVIEW: 'UPDATE_ADDON_REVIEW' = 'UPDATE_ADDON_REVIEW';
+export const DELETE_ADDON_REVIEW: 'DELETE_ADDON_REVIEW' = 'DELETE_ADDON_REVIEW';
+export const CLEAR_ADDON_REVIEWS_FOR_REVIEWID: 'CLEAR_ADDON_REVIEWS_FOR_REVIEWID' =
```

Comment:
I agree with changing `CLEAR` to `UNLOAD`. I guess I went with `CLEAR` because I was replacing another action that was already called `CLEAR`. That's also part of the reason I used `CLEAR_ADDON_REVIEWS_FOR_REVIEWID`, because originally I wasn't sure that we could just remove `CLEAR_ADDON_REVIEWS` so I needed the name to be different.

I don't think `UNLOAD_ADDON_REVIEW` makes sense though, because what we're doing is not unloading an addon review; what we're doing is unloading any caches of reviews which contained this review, which I think is a bit different. Maybe just `UNLOAD_ADDON_REVIEWS` would be okay for that, but I kind of like the `FOR_REVIEWID` part because it makes it more explicit exactly what we're doing. Either works for me though. What do you think?

Reply:
I'd go for `UNLOAD_ADDON_REVIEWS` then :)